### ISSUE TITLE
already prepared error

### ISSUE THREAD

**Issue by athoune** [factorysh/microdensity#37]
https://github.com/factorysh/microdensity/blob/events/queue/queue_test.go#L84-L92

```
--- FAIL: TestDeq (1.23s)
    /Users/mlecarme/Developpement/bearstech/microdensity/queue/queue_test.go:86: 
        	Error Trace:	queue_test.go:86
        	Error:      	Received unexpected error:
        	            	task with id `17e85e28-3ec3-4bcc-ade4-d1756812b747` already prepared
        	Test:       	TestDeq
    /Users/mlecarme/Developpement/bearstech/microdensity/queue/queue_test.go:88: 
        	Error Trace:	queue_test.go:88
        	Error:      	Received unexpected error:
        	            	task with id `ed3afe37-b533-4169-951d-3cbdee6197b7` already prepared
        	Test:       	TestDeq
    /Users/mlecarme/Developpement/bearstech/microdensity/queue/queue_test.go:90: 
        	Error Trace:	queue_test.go:90
        	Error:      	Received unexpected error:
        	            	task with id `5563b04d-88c9-49ba-bcc4-72034bc041cd` already prepared
        	Test:       	TestDeq
FAIL
FAIL	github.com/factorysh/microdensity/queue	1.698s
FAIL
```

**Comment by papey** [comment]
Can't reproduce : 

```
Running tool: /usr/lib/go/bin/go test -timeout 30s -run ^TestDeq$ github.com/factorysh/microdensity/queue

ok  	github.com/factorysh/microdensity/queue	1.477s
```

**Comment by papey** [comment]
Fixed by #25
